flake.nix

```diff
@@ -186,6 +209,12 @@
             inherit fileset stdenv;
           };
 
+          # https://github.com/NixOS/nixpkgs/pull/214409
```

Comment:
What does that mean? The PR top post doesn't state the problem it's supposedly solving